stories/core/Buttons/ButtonGroup.stories.tsx

```diff
@@ -37,3 +54,61 @@ export const WithIcons: Story<ButtonGroupProps> = () => {
     </ButtonGroup>
   );
 };
+
+export const Overflow: Story<ButtonGroupProps> = (args) => {
+  const buttons = Array(10)
+    .fill(null)
+    .map((_, index) => (
+      <IconButton
+        key={index}
+        onClick={() => action(`Clicked on button ${index + 1}`)()}
+      >
+        <SvgPlaceholder />
+      </IconButton>
+    ));
+
+  return (
+    <div
+      style={{
+        maxWidth: 'clamp(300px, 50%, 100%)',
+        border: '1px solid hotpink',
+        padding: 8,
+      }}
+    >
```

Comment:
I would add just a comment above to change the size to see it working or smth..

Reply:
I really like Blueprint breadcrumbs demo:
![blueprint-breadcrumbs](https://user-images.githubusercontent.com/36186912/129860152-39caa75f-6238-4912-a6b8-60cd5acc0bcd.gif)
https://blueprintjs.com/docs/#core/components/breadcrumbs

Reply:
Adding Slider was too much code so I just added text